assets/components/istio-operator/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
```

Comment:
Should we be consistent and put `experimental-` prefix in package name, assets directory etc?

Reply:
I thought it was meant only for the UX. Internally things stay the same, no?